src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -706,15 +674,35 @@ else if(world.getEnvironment() == Environment.NORMAL)
         //default siege blocks
         this.config_siege_blocks = new ArrayList<Material>();
         this.config_siege_blocks.add(Material.DIRT);
+        this.config_siege_blocks.add(Material.GRASS_BLOCK);
```

Comment:
I think this is supposed to be TALL_GRASS not GRASS_BLOCK. The previous version had LONG_GRASS which has been replaced by TALL_GRASS. I only noticed this because I made the same mistake at first editing this same exact line.

Reply:
Here I changed GRASS to GRASS_BLOCK and LONG_GRASS to GRASS. What used to be DOUBLE_PLANT with data of 2 is now TALL_GRASS in 1.13.

I don't think changes here are needed (though adding TALL_GRASS seems sensible, it wasn't included originally and my intention was purely to update this for 1.13 without any other changes)

Reply:
My guess is that only material IDs were able to be put in this list, so if the ID for `TALL_GRASS` was present before, then the material(s) that it represented should be included as well, not just the first with data type 1.


Unless I'm not understanding what you said here.

Reply:
Yes, thats what I meant.. I mixed up TALL_GRASS and LONG_GRASS in my initial comment to zedwick. I did end up adding all 3 new material names in my fork after realizing my mistake. https://github.com/treydun/GriefPrevention/blob/1b894abe04c687f7c698fd7d642ce182b538c4b9/src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java#L700 I added them in a dedicated final enum class that I have been using to update all the plugins we use. https://github.com/treydun/GriefPrevention/blob/1b894abe04c687f7c698fd7d642ce182b538c4b9/src/main/java/com/buildatnight/utils/Untagged.java It is materials that are still lacking a grouping tag in spigot. I realize this is not ideal but I was tired of typing the same list of 16-17 colored material items out or 6 wood types of everything.

Reply:
Sorry, was referring to @Zedwick's response, but as I re-read I might've been confused by your response as well. Is there any mapping to what the old/new Material enums are to the Minecraft blocks before I end up confused as well... a quick search doesn't seem to reveal much about `LONG_GRASS` other than the `LongGrass` class, which idk if that's new or not.

Reply:
@RoboMWM No problem! The only mapping from old -> new materials is that spigot renamed old Material names to LEGACY_(Old material Name) That only helps plugins that are 1.12 and cant/wont be upgraded. Its strongly advised, especially once you put api-version: 1.13 in your plugin.yml to only use the new names. As far as Item names changing goes, on the vanilla end I use https://minecraft.gamepedia.com/1.13/Flattening That is the list of all name changes from mojangs perspective. Spigot/Bukkit also changed their material names to better match mojangs. The full list of current Material names in the bukkit enumset is here. https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Material.html To help with grouping of items/blocks (because they all have individual names and dedicated classes) spigot also introduced the new Tag system. https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Tag.html# The items are tagged at the item registry level and then the Tag class in bukkit lets you pull enumsets of items tagged with whatever the enumsets name is. For example, Tag.SAPLINGS.getValues() is all the saplings Materials. You can also check if(Tag.SAPLINGS.isTagged(someitem)) Its not well documented yet and I havent figured out how to extend and tag my own items yet. Thats why I am using a helper enum currently

Reply:
Oh I see, `GRASS` now refers to that specific variant of the one block high, passable stuff. Not the two block high stuff. I thought that was considered tall grass all this time... 

Anyways, its variants (dead bush and fern) should also be added then for a "true" 1.13 conversion.

Reply:
Yes, exactly. That was my confusion as well